blazar/tests/manager/test_enforcement.py

```diff
@@ -99,9 +99,9 @@ def test_new_lease_against_default_limit(self):
     def test_new_lease_against_project_limits(self):
         self.cfg.CONF.set_override('default_max_lease_duration', 86400,
                                    group='enforcement')
-        self.enforcement.project_max_lease_durations = {
-            'project_bar': 86400 * 2,
-        }
+        self.exempt_project = self.patch(self.enforcement,
```

Comment:
What if you just use `set_override` and add the project as exempt? Then you are also testing that the conf mechanism works.

Reply:
Yep yep. I've updated and add comments in this so it's more clear what is being tested.